cmd/crc/cmd/status.go

```diff
@@ -68,12 +69,13 @@ func runStatus(writer io.Writer, client client, cacheDir, outputFormat string) e
 		return fmt.Errorf("Error finding size of cache: %s", err.Error())
 	}
 	status := Status{
-		CrcStatus:       clusterStatus.CrcStatus,
-		OpenShiftStatus: clusterStatus.OpenshiftStatus,
-		DiskUsage:       clusterStatus.DiskUse,
-		DiskSize:        clusterStatus.DiskSize,
-		CacheUsage:      size,
-		CacheDir:        cacheDir,
+		CrcStatus:        clusterStatus.CrcStatus,
+		OpenShiftStatus:  clusterStatus.OpenshiftStatus,
+		OpenShiftVersion: clusterStatus.OpenshiftVersion,
+		DiskUsage:        clusterStatus.DiskUse,
+		DiskSize:         clusterStatus.DiskSize,
+		CacheUsage:       size,
+		CacheDir:         cacheDir,
```

Comment:
This is again public facing UX change so I would like to have discussion about it before merge this one.

Reply:
The status message didn't change. With json output, for sure, now we cannot change it like this.

But for now, I don't think someone is relying on this struct ? Does someone vendor crc ?

Reply:
> Does someone vendor crc ?

@guillaumerose not that I am aware of.

Reply:
The trays depend on the struct, so will need changes there

Reply:
Yes, looking at the macOS tray we use CrcStatus, OpenshiftStatus, DiskUse and DiskSize :
```
➜  tray-macos git:(master) rg "status\."
CodeReady Containers/DetailedStatusViewController.swift
44:                    if status.Success {
45:                        self.vmStatus.stringValue = status.CrcStatus
46:                        self.ocpStatus.stringValue = status.OpenshiftStatus
47:                        self.diskUsage.stringValue = "\(Units(bytes: status.DiskUse).getReadableUnit()) of \(Units(bytes: status.DiskSize).getReadableUnit()) (Inside the VM)"
```

This PR (apart from breaking the json due the case) removes the version from OpenshiftStatus. 

What should we do ? The tray is still experimental so we can edit in both and release at the same time. WDYT of the case of the json ? I took the same convention as kubernetes/openshift.

Reply:
In fact this struct is not used by the tray. This struct is only use in cmd package. Handler use ClusterStatusResult. So we are good.

Reply:
Yeah, sorry.. i thought i commented on the `ClusterStatusResult` structure changes, we can change in both, that's no issue. and i am fine with the case of the json, although i find Upper camel case more good to look at..